DataImport.Web/Features/Agent/EditAgent.cs

```diff
@@ -69,13 +71,13 @@ public Task<AddEditAgentViewModel> Handle(Query request, CancellationToken cance
                         vm.EncryptionFailureMsg = Constants.AgentDecryptionError;
                     }
                 }
-
+                vm.ActionFileCode = vm.ActionFileCode ?? AgentActionsFile.DeleteOnSuccessful.ToString();
```

Comment:
@mjimdev this looks like it will default the agent action to delete on success if a value isn't already set. *I think* that this is different than current functionality? The default should be whatever Data Import would do w/o this change

Reply:
@stevenarnold-ed-fi  Before the change, the functionality was Delete on success ( delete if "FileStatus.Loaded")